src/sagemaker/experiments/run.py

```diff
@@ -29,6 +29,7 @@
 from sagemaker.experiments._helper import (
     _ArtifactUploader,
     _LineageArtifactTracker,
+    _DEFAULT_ARTIFACT_PREFIX,
```

Comment:
Why we are using UpperCase, should we keep consistent with exist parameter?

Reply:
`_DEFAULT_ARTIFACT_PREFIX` is not a new constant introduced in this commit. It is an existing constant defined in `src/sagemaker/experiments/_helper.py`